test/built-ins/Math/f16round/not-a-constructor.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2024 Kevin Gibbons. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-ecmascript-standard-built-in-objects
+description: >
+  Math.f16round does not implement [[Construct]], is not new-able
+includes: [isConstructor.js]
+features: [Float16Array, Reflect.construct]
+---*/
+
+assert(!isConstructor(Math.f16round), "Math.f16round is not a constructor");
+
+assert.throws(TypeError, function () {
+  new Math.fround();
```

Comment:
I just noticed this while working on the Ladybird browser project, isn't this supposed to be `new Math.f16round()` instead of `new Math.fround()`?

Reply:
Yes, good catch. Do you want to make a PR?

Reply:
Sure! If someone else beats me to it, no issue either way.